<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="gb_ReceivedData">
      <property name="title">
       <string>Принятые данные</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="te_result">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Clear">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Таймер</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_Start">
         <property name="text">
          <string>Начать передачу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Stop">
         <property name="text">
          <string>Остановить передачу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_DataToSend">
      <property name="title">
       <string>Отправка данных</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_TextToSend">
         <property name="text">
          <string>Текст</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_TextToSend"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="pb_Send">
         <property name="text">
          <string>Отправить датаграмму</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pb_Start</tabstop>
  <tabstop>pb_Stop</tabstop>
  <tabstop>le_TextToSend</tabstop>
  <tabstop>pb_Send</tabstop>
  <tabstop>te_result</tabstop>
  <tabstop>pb_Clear</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_Clear</sender>
   <signal>clicked()</signal>
   <receiver>te_result</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
